<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreenShotsWindow</class>
 <widget class="QMainWindow" name="MainScreenShotsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>截图工具主窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>截图动作编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_screenshots">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>执行截图</string>
    </property>
   </widget>
   <widget class="QListView" name="scrLogicListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>281</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="printListView">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>256</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0">
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="dTextBtn">
       <property name="text">
        <string>查找页面元素</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="printBtn">
       <property name="text">
        <string>输出</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="dSwipeBtn">
       <property name="text">
        <string>滑动具体值</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="sleepBtn">
       <property name="text">
        <string>睡</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="dClickBtn">
       <property name="text">
        <string>点击指定位置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
